src/dotnet/commands/dotnet-add/dotnet-add-package/Program.cs

```diff
@@ -143,7 +143,7 @@ private string[] TransformArgs(string packageId, string tempDgFilePath, string p
 
             args.AddRange(_appliedCommand
                 .OptionValuesToBeForwarded()
-                .SelectMany(a => a.Split(' ')));
+                .SelectMany(a => a.Split(' ', 2)));
```

Comment:
I don't know the code pretty well, but what happens if I run:

`dotnet add package packageName --package-directory "D:\Cloud\Dropbox\codes\dotnet samples\nugets" --interactive`

or

`dotnet add package packageName --package-directory "D:\Cloud\Dropbox\codes\dotnet samples\nugets" --no-restore`

Reply:
That should be fine.  `OptionValuesToBeForwarded` returns an `IEnumerable<string>` for each option, so `SelectMany` should get values like `--package-directory c:\foo bar`, `--interactive`, etc.

Thus, the resulting `argv` will be [`--packge-directory`, `c:\foo bar`, `--interactive`].  Options with spaces in them, like `c:\foo bar` is then quoted depending on the platform rules by other CLI code.

Reply:
Ok that makes sense. 
And there are no options that take more than 1 value as far as I can tell.

This seems fine in that case.